core/templates/dev/head/pages/exploration_player/LearnerViewInfo.js

```diff
@@ -114,6 +116,8 @@ oppia.controller('LearnerViewInfo', [
               expInfo.last_updated_msec);
             $scope.numViews = expInfo.num_views;
             $scope.objective = expInfo.objective;
+            $scope.isExplorationPrivate = (
+              expInfo.status === 'private' ? true : false);
```

Comment:
You can just do (expInfo.status === 'private')

Reply:
Yeah obviously!